lib/chef/knife/vault_remove.rb

```diff
@@ -70,6 +70,7 @@ def run
               end
 
               remove_items.each do |key|
+                key = key.dup
                 key.strip!
                 vault_item.remove(key)
```

Comment:
Alternative offered from @thommay, instead of duplicating the key to `strip!` it in-place, use the return value of `key.strip` within the call to `remove()`:

```diff
- key.strip!
- vault_item.remove(key)
+ vault_item.remove(key.strip)
```